packages/apps-config/src/api/spec/crab.ts

```diff
@@ -1,6 +1,20 @@
 // Copyright 2017-2024 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { typesBundleForPolkadotApps } from '@darwinia/types/mix/index.js';
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
 
-export default typesBundleForPolkadotApps.spec.Crab;
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+const definitions: OverrideBundleDefinition = {
```

Comment:
Did you want these empty because you intend on adding types in the future?

If I understand it correctly since all the types are already available in the Metadata and there are no overrides you need, then you can just remove these files. To be sure though, I would ensure some manual testing :)

Reply:
That being said, are there no historical types here you would want?

Reply:
Yes, we've done a re-genesis before. It's a new chain using metadata v14 from block 0.

Reply:
Ahh okay thanks for the explanation!